lib/hammer_cli_foreman_remote_execution/job_invocation.rb

```diff
@@ -30,6 +30,8 @@ def extend_data(invocation)
 
     class InfoCommand < HammerCLIForeman::InfoCommand
       extend WithoutNameOption
+      option '--host-status', :flag, N_('Show job status for the hosts.')
```

Comment:
Description shouldn't end with a dot.

This is not actually needed here if it was specified as a param in the Foreman plugin (which I presume was: https://github.com/stejskalleos/foreman_remote_execution/blob/20491e542b229d02335bb8112eefc636814b136e/app/controllers/api/v2/job_invocations_controller.rb#L21). After removal It works, but if I explicitly provide value for `--host-status`, e.g. `--host-status true`.

This can be used to override the default behaviour. In this case we can leave it, but to get better UX, we could use here boolean normalizer to require a user to pass boolean values for this option. Example: https://github.com/theforeman/hammer-cli-foreman/blob/748d619d5e0e4219b705abcd9fee305c61f2d4b6/lib/hammer_cli_foreman/smart_class_parameter.rb#L119-L120.

Although it would be better if we change it here: https://github.com/stejskalleos/foreman_remote_execution/blob/20491e542b229d02335bb8112eefc636814b136e/app/controllers/api/v2/job_invocations_controller.rb#L21, i.e. change `String` to `:boolean`.

Both ways are pretty acceptable :)